bluefog/common/mpi_context.cc

```diff
@@ -344,6 +394,7 @@ bool MPIContext::RegisterWindowName(const std::string& name) {
   }
   auto win_manager_ptr = std::make_shared<WindowManager>();
   win_manager_ptr->InitializeMutexWin(mpi_comm);
+  win_manager_ptr->InitializeVersionWin(mpi_comm, neighbor_in_ranks_);
```

Comment:
Add assert to wrap this. 
assert(win_manager_ptr->InitializeVersionWin(mpi_comm, neighbor_in_ranks_));

Do the same thing for InitializeMutexWin and InitializePWin as well.